--- __init__.py	2019-12-12 22:35:52.321163000 +0800
+++ tmp/__init__.py	2019-12-16 18:32:01.226283000 +0800
@@ -230,7 +230,8 @@
 
 import django.conf
 from django.core import exceptions
-from django.core import urlresolvers
+# from django.core import urlresolvers
+from django.urls import reverse
 from six.moves.urllib import parse
 
 from oauth2client import clientsecrets
@@ -408,7 +409,7 @@
     Returns:
         A properly formatted redirect string.
     """
-    url = urlresolvers.reverse(url_name, args=args)
+    url = reverse(url_name, args=args)
     params = parse.urlencode(kwargs, True)
     return "{0}?{1}".format(url, params)
 
